pyce/_imports.py

```diff
@@ -164,3 +164,17 @@ def find_spec(cls, fullname: str, path: Optional[List[str]] = None,
             if sorocospec is not None:
                 break
         return sorocospec
+
+
+def add_import_hook(keys={}):
```

Comment:
Don't have a mutable default.

https://docs.python-guide.org/writing/gotchas/

Reply:
Thanks @pradyunsg!  I seem to have regressed this morning.  Going to fix that after this cup of coffee.

I'm also adding type hints.